lms/envs/common.py

```diff
@@ -2118,14 +2118,16 @@
 
 # JWT Settings
 JWT_AUTH = {
-    'JWT_SECRET_KEY': None,
+    # TODO Set JWT_SECRET_KEY to a secure value. By default, SECRET_KEY will be used.
+    # 'JWT_SECRET_KEY': '',
```

Comment:
To clarify, I expect anyone issuing JWTs to override this, hence the need to update docs/release notes.